webscraping/webscrape.py

```diff
@@ -0,0 +1,47 @@
+import http.client
+
+def firstBetween (string, startingString, endingString): #This function return whatever is between the starting string and ending string based off of an original string
+    idx = [i for i in range(len(string)) if string.startswith(startingString, i)]
+    if len(idx) > 0:
+        ret = exerciseHTML[idx[0]+ len(startingString):].split(endingString)[0]
```

Comment:
I don't think we can use exerciseHTML here directly?
Also maybe we can simplify this function a little bit using FIND. Something like:
```
def find_between(text, start, end):
    start_idx = text.find(start)
    if start_idx == -1:
        return None
    start_idx += len(start)
    end_idx = text.find(end, start_idx)
    if end_idx == -1:
        return None 
```

Reply:
Good catch! Works just as well with this version